src/main/java/jnr/posix/LinuxFileStat64.java

```diff
@@ -10,21 +10,22 @@ public Layout(jnr.ffi.Runtime runtime) {
         }
 
         public final Signed64 st_dev = new Signed64();
-        public final Signed64 st_ino = new Signed64();
+        public final Unsigned64 st_ino = new Unsigned64();
         public final Signed64 st_nlink = new Signed64();
-        public final Signed32 st_mode = new Signed32();
-        public final Signed32 st_uid = new Signed32();
-        public final Signed32 st_gid = new Signed32();
+        public final Unsigned32 st_mode = new Unsigned32();
+        public final Unsigned32 st_uid = new Unsigned32();
+        public final Unsigned32 st_gid = new Unsigned32();
+        public final Signed32 __pad0 = new Signed32();
```

Comment:
Originally I tried on `Debian 8.8 / cc (Debian 4.9.2-10) 4.9.2 / Xeon E5620`. At the moment I rechecked on `CentOS 7.3.1611/ cc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-11) / i7-4770HQ`.

And also using clang on `CentOS` (`clang version 3.4.2 (tags/RELEASE_34/dot2-final)`) and got the same result everywhere.

P.S. I'm not sure but one C dev who I know said that this should be guaranteed by C ABI.